### DISCUSSION TITLE
Cannot run exe from wsl2 after enabled systemd support

### DISCUSSION THREAD

**Discussion by newini** [microsoft/WSL#9755] [General] [answered]
### Windows Version

Microsoft Windows [Version 10.0.22621.1265]

### WSL Version

1.0.3.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.79.1

### Distro Version

Ubuntu 20.04

### Repro Steps

1. Enable systemd support on WSL2. Following instruction [here](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#systemd-support)

2. Run any Windows applications on WSL2. e.g. type:
```
explorer.exe .
```

### Expected Behavior

Expected to open explorer.exe on Windows

### Actual Behavior

There is an error message:
```
run-detectors: unable to find an interpreter for /mnt/c/Windows/explorer.exe
```

**Comment by klabacher** [comment]
Hello Sir, maybe this is a duplicate from my previous error. Take a look at this [Issue](https://github.com/microsoft/WSL/issues/9381)

**Comment by ghost** [comment, marked as answer]
Your binfmt_misc registration for interop have been overwritten. You can use `sudo update-binfmts --disable [NAME-HERE]` to disable whatever is overwriting this. Example of a problematic interpreter here:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/102559858/223568302-3cb5c3b9-cd3a-42d1-a192-59e202543313.png">

**Reply by newini** [reply]
@pmartincic Thank you so much! I fixed the problem by disabled 'cli'.
```
sudo update-binfmts --disable cli
```
![image](https://user-images.githubusercontent.com/18367015/223880345-bae94008-77d5-4e26-8f87-e6b877bc9203.png)

Now I can run Windows app from WSL2 again while enabled systemd support.
